app/views/users/_profile.html.erb

```diff
@@ -70,3 +81,23 @@
     <button type="submit" class="crayons-btn">Save Profile Information</button>
   </div>
 <% end %>
+
+<script>
```

Comment:
it'd be great if these instructions went in their own JS file so that they can be linted and compressed.

BTW I think we should start using event bubbling to have a single event handler on the immediate container instead of having N event handlers for each element but that's a larger conversation for another day and for another PR, outside the scope of this one. cc @nickytonline

Reply:
We have a file called `initializeSettings` where these could go.

I feel like as a hot fix we can accept this though and then make another change after.

Reply:
> it'd be great if these instructions went in their own JS file so that they can be linted and compressed.
> 
> 
> 
> BTW I think we should start using event bubbling to have a single event handler on the immediate container instead of having N event handlers for each element but that's a larger conversation for another day and for another PR, outside the scope of this one. cc @nickytonline 

@rhymes, definitely leverage event bubbling.